src/main/resources/anet-schema.yml

```diff
@@ -524,11 +524,48 @@ properties:
             additionalProperties:
               "$ref": "#/$defs/customField"
 
+      attachment:
+        type: object
+        additionalProperties: false
+        required: [shortLabel, shortName, description, classification]
+        properties:
+          disabled:
+            type: boolean
+            default: false
```

Comment:
This line doesn't do anything. You probably want to make this filed required instead.